delta/delta.go

```diff
@@ -104,6 +137,19 @@ func (d *Delta) Insert(text string, attrs map[string]interface{}) *Delta {
 	return d
 }
 
+// InsertEmbed takes a map of embeds and a map of attributes, adds them to the Delta. This
+// can be used to insert images or URLs to the Delta
+func (d *Delta) InsertEmbed(embeds map[string]interface{}, attrs map[string]interface{}) *Delta {
+	newOp := Op{
+		InsertEmbed: embeds,
+	}
+	if attrs != nil {
```

Comment:
I think we can skip the != nil check and we'll end up with the same result (we should add tests to make sure and avoid future breaking changes/bugs)